nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -479,6 +479,18 @@
      been removed.
     </para>
    </listitem>
+  <listitem>
+    <para>
+     The nginx web server previously started its master process as root
+     privileged, then ran worker processes as a less privileged identity. This
+     was changed to start all of nginx as a less privileged user (defined by
+     <literal>services.nginx.user</literal> and
+     <literal>services.nginx.group</literal>). As a consequence, all files that
+     are needed for nginx to run (included configuration fragments, SSL
+     certificates and keys, etc.) must now be readable by this less privileged
+     identity.
+    </para>
+   </listitem>
```

Comment:
I think we should move this to the next release (19.09), so it receives more testing.

Reply:
On the test machine starts normally

Reply:
Yes, but we are just 3 weeks away from 19.03 and nginx is quite heavily used in many different setups. So your test setup does not cover all use-cases.